management/azure_mgmt_analysis_services/lib/generated/azure_mgmt_analysis_services/models/analysis_services_server_update_parameters.rb

```diff
@@ -12,7 +12,6 @@ class AnalysisServicesServerUpdateParameters
 
       include MsRestAzure
 
-      include MsRest::JSONable
```

Comment:
why was this included before?

Reply:
It was a remnant from a serialization task which was later dropped.